dbt/adapters/athena/impl.py

```diff
@@ -57,6 +57,48 @@ def convert_number_type(cls, agate_table: agate.Table, col_idx: int) -> str:
     def convert_datetime_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "timestamp"
 
+    # TODO: Add more lf-tag unit tests when moto supports lakeformation
+    # moto issue: https://github.com/getmoto/moto/issues/5964
+    @available
+    def add_lf_tags(self, database: str, table: str = None, lf_tags: Dict[str, str] = None):
```

Comment:
it would be great to add lf_tags for columns as well

Reply:
Let's consider to add that in a follow up PR. Columns inherit tags from the table, we need a tag to overwrite the tags and add more tags.